### PR TITLE
landingPage Changes

### PR DESCRIPTION
**Pull request by preethasri**

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]
I like your sidebar , but I would like to say, there is glitch when I click on hamburger and sidebar opens and vice versa for closing. I think its bacause of transition time. It looks great But, you can improve it later.  

Or you can do is keep sidebar fixed for desktop/laptop view and in responsiveness(tablet/mobile)  you can use that hide-show of sidebar functionality.

**Review by Amandeep-sinha — APPROVED** [review]
looking awesome ! great to go ahead 
 - should add cursor : pointer to openBtn : hover pseudo class

File: landingPage/landingPage.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+.library-sidebar .closeBtn {
+    position: absolute;
+    top: 0;
+    right: 25px;
+    font-size: 30px;
+    margin-left: 50px;
+}
+
+.openBtn {
```
Comment (KoushikKintali) [inline landingPage/landingPage.css]: It would be better if you follow the same naming convention as other

like openBtn => open-btn

Comment (preethasri) [reply landingPage/landingPage.css]: will work on this guys . thank you so much for your previews

**Pull request merged**